RemotingSample/Assets/Scenes/Simple Lighting Settings.lighting

```diff
@@ -0,0 +1,63 @@
+%YAML 1.1
```

Comment:
Is there a reason this scene has custom lighting settings? I don't think we have these in any of our other scenes in this repo.

Between this "Simple Lighting Settings.lighting" and all the LightingData info in the Scenes/RemotingSampleScene, I think these can all get generated from changing some Unity settings, but I'm not sure where they are off the top of my head

Reply:
most of them are carried forward from the main test project Keaton. removed them, looks like this is not needed infact.